packages/clis/parser/game-files/def-parser.ts

```diff
@@ -909,3 +922,39 @@ function processRouteJson(obj: RouteSii): Map<string, Route> {
   }
   return routes;
 }
+
+function processMileageTargetJson(
+  obj: MileageTargetsSii,
+): Map<string, MileageTarget> {
+  const mileageTargets = new Map<string, MileageTarget>();
+  for (const [key, rawTarget] of Object.entries(obj.mileageTarget)) {
+    const token = assertExists(key.split('.')[1]);
+    let target: MileageTarget = {
+      token: token,
+      editorName: rawTarget.editorName,
+      defaultName: rawTarget.defaultName,
+      nameVariants: Array.isArray(rawTarget.names) ? rawTarget.names : [],
+      distanceOffset: rawTarget.distanceOffset,
+    };
+    // Some mileage targets are specified with a position, which we can use
+    // directly. For other mileage targets, a node uid is given, which we'll
+    // have to try to resolve later using data from map sector files.
+    if (rawTarget.position[0] != null) {
+      assert(rawTarget.position[1] != null && rawTarget.position[2] != null);
```

Comment:
If the schema guarantees that this array is either all numbers or all undefineds (see diff file), then this can look like:

```suggestion
    if (rawTarget.position.every(v => v != null)) {
```

Reply:
Ahyes.